README.md

```diff
@@ -2,7 +2,17 @@
 
 ## Purpose
 
-_... help fill this in!_
+* Define and maintain security policies and procedures for:
+  - the core Node.js project
+  - other projects maintained by the Node.js Foundation technical group
+* Define and maintain policies and procedures for the coordination of security concerns within the external Node.js open source ecosystem
```

Comment:
I would add, "help maintainers fix high-impact security vulnerabilities". We have huge technical skills in the team, and we might help maintainers in need. Maintaining OSS is hard, and having a place where they can ask for help might be _very_ interesting.